appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -596,7 +596,7 @@ private YaVersion() {
   // - TRENDLINE_COMPONENT_VERSION was introduced
   // For YOUNG_ANDROID_VERSION 228:
   // - Renamed LineOfBestFit to Trendline
-  public static final int YOUNG_ANDROID_VERSION = 228;
+  public static final int YOUNG_ANDROID_VERSION = 229;
```

Comment:
You can leave this as 228 and we will bump it when doing a release.

Reply:
Done